bindings_ffi/src/mls.rs

```diff
@@ -1866,6 +1866,12 @@ impl FfiConversation {
         ))
     }
 
+    pub fn is_conversation_message_disappearing_enabled(&self) -> Result<bool, GenericError> {
+        let settings = self.conversation_message_disappearing_settings()?;
+
+        Ok(settings.from_ns > 0 && settings.in_ns > 0)
```

Comment:
@mchenani So I think this logic is wrong because what you actually care about is if they are both equal to 0 or not. If one is zero and the other is 10 then it would not be disabled

Reply:
Having both zero, doesn't make sense, bc means messages from the beginning and live 0ns! At the end we let them live 1 second! Bc the worker run on 1 second cycle.